create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/utils/wait_script.py

```diff
@@ -1,17 +1,35 @@
-{%- if cookiecutter.use_postgres == 'y' %}
-
-import socket
+{%- if cookiecutter.use_postgres == 'y' or cookiecutter.use_redis == 'y' %}import socket
 import time
+import random
 
 from {{ cookiecutter.project_name }}.utils.common import get_config, DEFAULT_CONFIG_PATH
 
 if __name__ == '__main__':
-    postgres = get_config(['-c', DEFAULT_CONFIG_PATH.as_posix()])['postgres']
+    {%- if cookiecutter.use_postgres == 'y' or cookiecutter.use_redis == 'y' %}
```

Comment:
if we don't use postgres or redis we remove this file, so this `if` is not required.